src/BizHawk.Emulation.Cores/Consoles/Nintendo/GBHawk/GBHawk.cs

```diff
@@ -213,13 +215,13 @@ public GBHawk(CoreComm comm, GameInfo game, byte[] rom, /*string gameDbFn,*/ GBS
 			ser.Register<ISoundProvider>(audio);
 			ServiceProvider = ser;
 
-			_settings = (GBSettings)settings ?? new GBSettings();
+			_ = PutSettings(settings ?? new GBSettings());

```

Comment:
Why is this being changed?

Reply:
`PutSettings` updates the dissassembler.